masl.src.html

```diff
@@ -99,44 +99,44 @@ <h3>Single or Multiple Resources</h3>
             map are complete paths that MUST start with <code>/</code> and the values are metadata
             objects that MUST have an <code>src</code> field pointing to the resource's CID and SHOULD have a
             <code>mediaType</code> field giving its MIME type, along with other HTTP headers.
+            Any other properties present which do not map to HTTP headers MUST be ignored.
           </li>
         </ul>
         <p>
           Note that if both <code>src</code> and <code>resources</code> are specified, then
           <code>src</code> MUST be ignored.
         </p>
         <p>
-          The Bundle Mode has some specific processing rules:
-          Loading bundles: default, relative, no directory, note not zip
+          The Bundle Mode has some specific processing rules when
+          loading bundles, which can be abbreviated to: default, relative, no directory, note not zip
         </p>
         <ul>
           <li>
-            The entry with path <code>/</code> is the default path that is loaded if the bundle
-            itself is being rendered. Implementations MUST only recognise this as the default
+            Default: The entry with path <code>/</code> is the default path that is loaded if the bundle
+            itself is dereferenced for rendering, e.g. in a browser or other user-agent. Implementations MUST only recognise this as the default
```

Comment:
```suggestion
            itself is used for rendering, e.g. in a browser or other user agent. Implementations MUST only recognise this as the default
```

Reply:
a rendering engine turning a reference into an object to be rendered is a pretty clear example of "dereferencing", while "to use" is the vaguest verb in the english language

Reply:
But that implies that there was dereferencing involved… If you double-click on a tile and it opens in your UA, no dereferencing ever happened!

Reply:
```suggestion
            itself gets rendered, e.g. in a browser or other user-agent. Implementations MUST only recognise this as the default
```